docs/en/reference/introduction.rst

```diff
@@ -1,17 +1,15 @@
 Introduction
 ============
 
-The Doctrine database abstraction & access layer (DBAL) offers a
-lightweight and thin runtime layer around a PDO-like API and a lot
-of additional, horizontal features like database schema
-introspection and manipulation through an OO API.
-
-The fact that the Doctrine DBAL abstracts the concrete PDO API away
-through the use of interfaces that closely resemble the existing
-PDO API makes it possible to implement custom drivers that may use
-existing native or self-made APIs. For example, the DBAL ships with
-a driver for Oracle databases that uses the oci8 extension under
-the hood.
+The Doctrine database abstraction & access layer (DBAL) offers an
```

Comment:
This is the first time I see it unabbreviated as "abstraction & access". Let's keep it as a "database abstraction layer".

Reply:
Me too :-)